website/blog/2019-06-04-hello-reason-react-native.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 2019-06-04-hello-reason-react-native
+title: Hello Reason React Native
+---
+
+Today is a big day for the ReasonML community as we are releasing the fruits of
+a long effort started 3 months ago. As more and more people were starting to use
+ReasonML on production apps using React Native, we wanted to make the
+BuckleScript bindings for React Native, known has `bs-react-native`, more
+accessible. This would have involve several breaking changes in order to
+simplify some APIs that have been designed with a high level of security in
+mind, but at a some costs. The cost of being harder to start with (for newcomers
+to ReasonML) as well as the cost in terms of code size and performance.
+
+While some of us were thinking about some simplifications, some others, involved
+in Reason React incoming changes, directly started a new approach for this
+bindings, following
+[the zero-cost approach offered by reason-react 0.7.0](https://reasonml.github.io/reason-react/blog/2019/04/10/react-hooks).
+
+[CCA](https://www.cca.io) started this effort on what is now know as the new
+`reason-react-native` bindings, that remove most of the additional JavaScript
+code produced by the bindings.
+
+## Zero-cost
+
+We mean almost zero additional cost. So the code you produce with the bindings
+will most of the time directly hit React Native JavaScript without an additional
+cost on top of it.
```

Comment:
`By zero-cost, we means zero additional cost. The code you produce using this bindings
will most of the time directly hit React Native JS code without creating an unnecessary abstraction above it 🙌`